app/src/main/java/com/T05/krowdtrialz/util/ExperimentList.java

```diff
@@ -44,7 +52,27 @@ public View getView(int position, @Nullable View convertView, @NonNull ViewGroup
         experimentOwner.setText(experiment.getOwner().getUserName().toString());
         experimentDescription.setText(experiment.getDescription().toString());
         experimentStatusCheck.setChecked(experiment.isActive());
-        
+
+        view.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                Intent intent;
+
+                if(experiment.isOwner(deviceUser)){
+                    // If user is owner
+                    intent = new Intent(context, ExperimentDetailsOwnerActivity.class);
+                } else {
+                    // If user is not owner
+                    intent = new Intent(context, ExperimentDetailsNonOwnerActivity.class);

```

Comment:
@arrooney I guess you'll eventually have to change this since these will be a single activity.